### PR TITLE
Refactor/bits and pieces

### PR DESCRIPTION
**Pull request by knees4bees**
# PR PlanetParty
**Connor A-L, Alex T & Katie B**

### What does this do?

- [x] Fix
- [ ] Feature
- [x] Refactor
- [x] Style
- [ ] Testing

### Are there any known bugs?

- [x] No
- [ ] Yes (if so please disclose in Notes for Reviewer & Next Iterations)

### Summary of the changes
- Fixes styling bugs with header component
- Styles gravity input and output
- Adds functionality to scroll to top on mount

### What ticket(s) do the changes resolve?
#64, #67 (and decides not to complete #73)
Please Link Issues to this PR as necessary.

### How to Test Changes?
Please check styling on planet detail page and confirm that spacing and responsiveness are still working well.

### CODE REVIEW

File: src/components/App/ScrollToTopOnMount.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { useEffect } from "react";
```
Comment (alexthompson207) [inline src/components/App/ScrollToTopOnMount.tsx:1]: This is awesome, way to find a solution to a not-so-obvious problem!!! And it's super clean!

File: src/components/PlanetInfo/planetInfo.css [hunk 2]
```diff
@@ -8,6 +8,10 @@ Light Blue: #68CCD7
 Yellow: #F5AE45
 Light Gray: #D0D7DF */
 
+* {
```
Comment (alexthompson207) [inline src/components/PlanetInfo/planetInfo.css:11]: Nice! Our app is looking super accessible!

**Review by alexthompson207 — APPROVED** [review]

File: src/components/App/ScrollToTopOnMount.tsx (hunk 1 above)
Comment (ConnorAndersonLarson) [reply src/components/App/ScrollToTopOnMount.tsx:1]: Seconded, good find!

**Review by ConnorAndersonLarson — APPROVED** [review]
Great, thanks for getting all these little things out of the way!

**Pull request merged**